kibbeh/public/locales/ro/translation.json

```diff
@@ -37,11 +37,11 @@
       "currentRoom": "Deocamdată în:",
       "startPrivateRoom": "Începe o cameră privată cu ei"
     },
-    "followList": { "followHim": "follow", "followingHim": "following" },
+    "followList": { "followHim": "follows", "followingHim": "following" },
```

Comment:
Is the extra `s` the actual translation?

Reply:
I think that is a mistake

Reply:
how do I change it?

Reply:
You can change your pull-request by looking at files in your [fork](https://github.com/LuisPavelA/dogehouse/tree/patch-1). The file you are looking for is [here](https://github.com/LuisPavelA/dogehouse/blob/patch-1/kibbeh/public/locales/ro/translation.json). You can edit it in the browser, or if you have cloned the fork to your computer, you can edit it there and push to your fork.

Reply:
I can't find the mistake in my fork :(

Reply:
Yeah it appears there isn't a mistake like that on the fork on my end either.

Reply:
The most recent commit appears to resolve this.